test/draw/RasterCanvasTest.ooc

```diff
@@ -3,6 +3,7 @@ use ooc-math
 use ooc-draw
 use ooc-draw-gpu
 use ooc-opengl
+import RasterCanvas
```

Comment:
This will be gone later, for now I just want to make sure that the backend method works correctly.